src/dict.h

```diff
@@ -207,6 +208,7 @@ dictEntry *dictTwoPhaseUnlinkFind(dict *d, const void *key, dictEntry ***plink,
 void dictTwoPhaseUnlinkFree(dict *d, dictEntry *he, dictEntry **plink, int table_index);
 void dictRelease(dict *d);
 dictEntry * dictFind(dict *d, const void *key);
+dictEntry * dictFindByHash(dict *d, const void *key, uint64_t hash);
```

Comment:
```suggestion
dictEntry *dictFindByHash(dict *d, const void *key, uint64_t hash);
```

Reply:
There is another similar function:
```
dictEntry *dictFindEntryByPtrAndHash(dict *d, const void *oldptr, uint64_t hash);
```
We can simplify its name and align it to:
```
dictEntry *dictFindByHashAndPtr(dict *d, const void *oldptr, uint64_t hash); 
```
And group it together in file dict.h next to `dictFind` and `dictFindByHash`

Reply:
addressed in 44b45c6d17d1948bac5cbd00046a566e67d57bc9